tests/agent/test_local_agent.py

```diff
@@ -87,18 +87,22 @@ def test_local_agent_uses_ip_if_dockerdesktop_hostname(monkeypatch):
     assert "IP" in agent.labels
 
 
-def test_populate_env_vars():
+def test_populate_env_vars(monkeypatch):
     agent = LocalAgent()
 
+    # The python path may be a single item and we want to ensure the correct separator
+    # is added so we will ensure PYTHONPATH has an item in it to start
+    if not os.environ.get("PYTHONPATH", ""):
+        monkeypatch.setenv("PYTHONPATH", "foobar")
```

Comment:
You can use

```suggestion
monkeypatch.setenv("PYTHONPATH", "foobar", prepend=os.pathsep)
```

universally here, no need for conditional checking or comment.

Not a big deal though, feel free to ignore.